mkdocs.yml

```diff
@@ -26,6 +28,10 @@ plugins:
 markdown_extensions:
   - attr_list
   - md_in_html
+  - pymdownx.arithmatex
```

Comment:
Which Python package includes this extension? In general, can you also write instructions for building the docs?

For this, I recommend to move the docs dependencies from docs/requirements.txt into pyproject.toml as optional dependencies following [the example in the PDM docs](https://pdm-project.org/latest/reference/pep621/#optional-dependencies). Then the building the docs instructions can be as simple as
```
pdm install -G docs
mkdocs build
```